.github/workflows/c3-e2e-dependabot.yml

```diff
@@ -55,7 +60,7 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
         with:
-          fetch-depth: 0
```

Comment:
Note: `0` would also work but it seems quite unnecessary in my opinion since it fetches _everything_ (https://github.com/actions/checkout)

![Screenshot 2024-03-05 at 13 29 51](https://github.com/cloudflare/workers-sdk/assets/61631103/18d54364-b050-467c-8838-166b668eafb9)

I'm happy to use 0 if we find it more clear/robust and don't care about the extra fetching, please let me know what you think 🙂

Reply:
We've been going with 0 because it guarantees that all the tags are also there, and generally no one is complaining about performance. So I would err on that side. It is very confusing if something is missing from the cloned repo because the errors are a bit vague.

Reply:
I've changed it to `0` 🙂👍